src/tests/function/uuid.rs

```diff
@@ -0,0 +1,45 @@
+use crate::*;
+
+test_case!(uuid, async move {
+    let test_cases = vec![
+        (
+            "CREATE TABLE test (id INT UNIQUE NOT NULL)",
+            Ok(Payload::Create),
+        ),
+        ("INSERT INTO test (id) VALUES (1)", Ok(Payload::Insert(1))),
+        (
+            "SELECT id FROM test WHERE UUID() IS NOT NULL", // Unreliable result
+            Ok(select!(
+                id
+                Value::I64;
+                1
+            )),
+        ),
+        /*( Cannot test as INSERT is behaving poorly at the moment
+            "INSERT INTO test (id) VALUES (UUID()), (UUID()), (UUID()), (UUID()), (UUID())",
+            Ok(Payload::Insert(5)), // Should error if UUID isn't working
+        ),
+        (
+            "INSERT INTO test (id) VALUES (UUID())",
+            Ok(Payload::Insert(1)), // Should error if UUID isn't working
+        ),*/
+
+        /*( Cannot test as INSERT is behaving poorly at the moment
+            "CREATE TABLE test_default (id INT UNIQUE NOT NULL DEFAULT UUID())",
+            Ok(Payload::Create),
+        ),
+        ("INSERT INTO test_default (id) VALUES (NULL)", Ok(Payload::Insert(1))),
+        (
+            "SELECT id FROM test_default", // Unreliable result
+            Ok(select!(
+                id
+                Value::I64;
+                1
+            )),
+        ),*/
```

Comment:
It is not proper to add this kind of comments to codes.
We cannot track what we should do using comments.
Rather than using comments, please register issues by generalizing it.
Then not only for us, someone can work on this issue.

Reply:
Already logged and already working on it in a PR (#199), hence the comments.

Reply:
Ok good, then we don't have to include this commens in this pr.

Reply:
Why not?
They are tests that I want to have but cannot yet have as there is bugs in the way?

Reply:
> They are tests that I want to have but cannot yet have as there is bugs in the way?

However, you said they are already included in existing issues. Then we don't need to have using comments which are hard to track changes.

Reply:
I'll change them to state which issue they relate to.